e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -49,6 +50,11 @@ const utilisationReportPage = {
         cy.get(`[data-cy="payment-details-row--paymentId-${paymentId}-feeRecordId-${feeRecordId}"] td[data-cy="facility-id"]`),
     },
   },
+  utilisationTab: {
+    utilisationTable: {
```

Comment:
If there's only 1x table, this could just be `table` so it's `utilisationTab.table` instead of `utilisationTab.utilisationTable`?